src/ray/protobuf/gcs.proto

```diff
@@ -244,6 +244,8 @@ message TaskStateUpdate {
   optional string actor_repr_name = 11;
   // Worker Pid.
   optional int32 worker_pid = 12;
+  // Is task paused by debugger.
+  optional bool is_debugger_paused = 13;
```

Comment:
we don't need to pass this as GCS's filter since we are no longer pushing down the filter for debugger to GCS?

Reply:
I think we are here right? https://github.com/ray-project/ray/pull/41067/files#diff-619b019a6fa687fece5fdb2058396e7e959c39010af5c3b027208fa4b5cc7eaeR106

Reply:
sorry - i meant another place.